client/coral-admin/src/actions/comments.js

```diff
@@ -14,30 +14,30 @@ export const fetchModerationQueueComments = () => {
     dispatch({type: commentTypes.COMMENTS_MODERATION_QUEUE_FETCH_REQUEST});
 
     return Promise.all([
-      coralApi('/queue/comments/pending'),
+      coralApi('/queue/comments/premod'),
       coralApi('/queue/users/pending'),
       coralApi('/queue/comments/rejected'),
```

Comment:
I'm wondering why are we making 3 calls just to get the total comments :thinking: @jde @riley